.codecov.yml

```diff
@@ -1,32 +1,44 @@
 #
-# This codecov.yml is the default configuration for
-# all repositories on Codecov. You may adjust the settings
-# below in your own codecov.yml in your repository.
+# Codecov Docs:
+# https://docs.codecov.com/docs
+#
+# Codecov Report:
+# https://app.codecov.io/gh/regen-network/regen-ledger
```

Comment:
I recommend taking a look at this to see where we might want to improve tests. Also it looks like some tests are not running as expected, in particular I noticed message validation for `MsgBridge` and `MsgBridgeReceive` are showing up with poor coverage and I believe this has to do with the use of `t.Parallel()`. :eyes: 

https://app.codecov.io/gh/regen-network/regen-ledger/compare/1230/tree/x/ecocredit/core/msg_bridge.go
https://app.codecov.io/gh/regen-network/regen-ledger/compare/1230/tree/x/ecocredit/core/msg_bridge_receive.go

Reply:
I took care of these two in #1226.